main.py

```diff
@@ -1,23 +1,19 @@
-""" Leaphy compiler and minifier backend webservice """
+"""Leaphy compiler and minifier backend webservice"""
 
 import asyncio
 import base64
-import tempfile
-from os import path
 
-import aiofiles
 from fastapi import FastAPI, HTTPException
 from fastapi.middleware.cors import CORSMiddleware
 from groq import Groq
 from python_minifier import minify
 
 from conf import settings
-from deps.cache import code_cache, get_code_cache_key, library_cache
-from deps.logs import logger
+from models import Sketch, PythonProgram, Messages
+
+from deps.sketch import _install_libraries, _compile_sketch, startup
```

Comment:
these functions should not be private (_), since they are used outside the module.